InventoryInStorePickup/Model/SearchRequest/Area/SearchTerm/CountryParser.php

```diff
@@ -22,26 +23,44 @@ class CountryParser implements ParserInterface
      * @var Data
      */
     private $data;
+
     /**
-     * @var DelimiterParser
+     * @var DelimiterConfig
      */
-    private $parser;
+    private $delimiterConfig;
 
     /**
      * @param Data $data
-     * @param DelimiterParser $parser
+     * @param DelimiterConfig $delimiterConfig
      */
-    public function __construct(Data $data, DelimiterParser $parser)
+    public function __construct(Data $data, DelimiterConfig $delimiterConfig)
     {
         $this->data = $data;
-        $this->parser = $parser;
+        $this->delimiterConfig = $delimiterConfig;
     }
 
     /**
      * @inheritdoc
      */
     public function execute(string $searchTerm, DataObject $result): void
     {
-        $result->setData(self::COUNTRY, $this->parser->getCountry($searchTerm) ?? $this->data->getDefaultCountry());
+        $result->setData(self::COUNTRY, $this->getCountry($searchTerm) ?? $this->data->getDefaultCountry());
+    }
+
+    /**
+     * Get country from Search Term.
+     *
+     * @param string $searchTerm
+     *
+     * @return string|null
+     */
+    public function getCountry(string $searchTerm): ?string
```

Comment:
Are you sure this method should be public?

Reply:
Public -> private.